functional_tests/ft_estimate_npmetric_erf.R

```diff
@@ -3,7 +3,7 @@ m_d <- generate_syn_data(sample_size = 2000)
 pseudo_pop <- generate_pseudo_pop(m_d$Y,
                                  m_d$treat,
                                  m_d[c("cf1","cf2","cf3","cf4","cf5","cf6")],
-                                 ci_appr = "matching",
+                                 ci_appr = "weighting",
```

Comment:
Why do you change "matching" to "weighting" here?

Reply:
ft_* are functional tests. It is just an example.

Reply:
OK!